etp-backend/src/main/clj/solita/etp/security.clj

```diff
@@ -18,21 +18,21 @@
     (let [{:keys [data]} jwt-payloads
           email (:email data)
           cognitoid (:sub data)
-          whoami (whoami-service/find-whoami
-                  db
-                  {:email         email
-                   :cognitoid     cognitoid
-                   :henkilotunnus (:custom:FI_nationalIN data)
-                   :virtu         {:localid      (:custom:VIRTU_localID data)
-                                   :organisaatio (:custom:VIRTU_localOrg data)}})]
+          whoami-find-opts {:email         email
+                            :cognitoid     cognitoid
+                            :henkilotunnus (:custom:FI_nationalIN data)
+                            :virtu         {:localid     (:custom:VIRTU_localID data)
+                                           :organisaatio (:custom:VIRTU_localOrg data)}}
+          whoami (whoami-service/find-whoami db whoami-find-opts)]
       (if whoami
         (->> (cond-> whoami
                email (assoc :email email)
                cognitoid (assoc :cognitoid cognitoid))
              (assoc req :whoami)
              handler)
         (do
-          (log/error "Unable to find käyttäjä using email in data JWT")
+          (log/error "Unable to find kayttaja using email in data JWT"
+                     whoami-find-opts)
```

Comment:
miksi lokituksessa on "using email", kun juuri sillä ei varmaan koskaan pitäisi hakea

Reply:
Mites unable to find kayttaja using jwt + lokitetaan (gdpr whoami-find-opts)

Reply:
gdpr funktio muuttaa henkilotunnuksen 90xxxx-xxxx